src/Graphing.hs

```diff
@@ -209,6 +211,12 @@ shrinkRoots (Graphing gr) = Graphing $ foldr AME.shrinkSingle gr currentDirect
     currentDirect :: [Node ty]
     currentDirect = Set.toList $ AM.postSet Root gr
 
+hasVertex :: Ord ty => ty -> Graphing ty -> Bool
```

Comment:
[Optional] Avoid adding dead code, unless required by acceptance criteria or subsequent PR.

Reply:
Wes and I talked about this, and we decided to leave it in for future use. You're definitely right to flag this though.